ebin/rebar.app

```diff
@@ -52,7 +53,11 @@
                   compiler,
                   crypto,
                   syntax_tools,
-                  tools]},
+                  tools,
+                  diameter,
```

Comment:
Unless `dialyzer_plt_apps` overrides all apps, it must be called `dialyzer_extra_apps`. Updated the patch accordingly.